packages/react-scripts/package.json

```diff
@@ -89,10 +89,18 @@
     "fsevents": "^2.3.2"
   },
   "peerDependencies": {
+    "node-sass": ">=7",
     "react": ">= 16",
```

Comment:
Maybe also bump to >= 17?

Reply:
👍 
I guess we will use this for any old projects using 16 it will act as a forcing function!